.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,15 @@
+**Note: new features are to be opened against next, hotfixes against master.**
+
+### Summary
+
+SUMMARY_GOES_HERE
+
+### Full changelog
```

Comment:
As long as you guys are willing to conform to the conventional commit template, I would recommend using conventional changelog. It is super useful in handling all that grunt work.